### DISCUSSION TITLE
plays/bahnisikhadhar/basic-calculator

### DISCUSSION THREAD

**Discussion by giscus** [reactplay/react-play#1021] [Play Feedback]
# plays/bahnisikhadhar/basic-calculator

This is a Simple calculator where one can do addition, subtraction, multiplication, division. All clear and delete facility is also there. No external library is used to make this play

https://reactplay.io/plays/bahnisikhadhar/basic-calculator

**Comment by aghildev** [comment]
Nice One To Learn Basics Of React
